.circleci/config.yml

```diff
@@ -662,17 +688,20 @@ workflows:
             - build-nightly-buster-securedrop-workstation-config
       - build-nightly-buster-securedrop-workstation-grsec:
           requires:
-            - build-nightly-buster-securedrop-workstation-config
```

Comment:
Oh, was this one a bug? I wonder if that would have caused the `build-nightly-buster-securedrop-workstation-grsec` and `build-nightly-buster-securedrop-keyring` to sometimes run in different orders, and if that was ever an issue.
It makes sense to specify the order unambiguously :white_check_mark: